### PR TITLE
Major site improvement: Introducing our product suite

### PR DESCRIPTION
**Pull request by atironati**

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -44,8 +48,79 @@ body {
...
+  font-size: 22pt;
+  margin-top: 0px;
+  padding-bottom: 10px;
+}
+
+.product-description {
+  font-family: monospace;
+}
+
+.inspiration-left {
```
Comment (ColinRTaylor) [inline styles/styles.css]: Really not sure about this className, the etymology of the word 'inspiration' is 

> immediate influence of God or a god.

Are we to infer from your use of the word here, that you think these styles are god like? Or that this is the sort of CSS that a god would write?

Comment (atironati) [reply styles/styles.css]: You may infer that if you wish, but I draw inspiration from all aspects of life, not just some god. The beauty of a leaf as it rests in a puddle after a morning rain, the wisps of cloud as they are strewn across a night sky full of stars, the sound of a small mammal when it is safe with its kin, all these and more serve as inspiration for what I think we can all agree is god like in a way

Comment (ColinRTaylor) [reply styles/styles.css]: Please change this class to `.divineBeing-left`

**Review by ColinRTaylor — CHANGES_REQUESTED** [review]
please change

**Pull request merged**